Cargo.toml

```diff
@@ -92,7 +84,7 @@ tracing = { version = "0.1.40", default-features = false }
 image = { version = "0.25.0", default-features = false, features = ["png"] }
 tracing = { version = "0.1.40", default-features = false, features = ["log"] }
 tracing-subscriber = { version = "0.3.18", features = ["env-filter"] }
-winit = { path = ".", features = ["rwh_05"] }
+winit = { path = "." }
```

Comment:
Isn't the current (lib) crate always part of `[dev-dependencies]` by default? It looks like it was only added here to force the `rwh_06` feature.

```suggestion
```

Reply:
Ah, right!

Reply:
It did have some use after all, enabling default dependencies.
But I addressed this by using the [`required-features` field](https://doc.rust-lang.org/1.80.0/cargo/reference/cargo-targets.html#the-required-features-field).

Reply:
Ah, good catch! `requires-features` is a nicer one (though I do sometimes with Rust had a `cargo run --with-required-features --example ...` to simplify).